demos/storybook/package.json

```diff
@@ -8,10 +8,8 @@
     "storybook": "start-storybook -p 6006 -s ../../docs/,./assets,./node_modules/typeface-open-sans/files",
     "build-storybook": "build-storybook -s ../../docs/,./assets,./node_modules/typeface-open-sans/files",
     "lint": "eslint \"stories/**/**.ts\"",
-    "lint:fix": "eslint \"stories/**/**.ts\" --fix",
-    "prettier": "prettier \"stories/**/**.{ts,json,css,scss,html}\" --write"
+    "lint:fix": "eslint \"stories/**/**.ts\" --fix"
   },
-  "prettier": "@pxblue/prettier-config",
```

Comment:
Why was prettier removed from the storybook?

Reply:
I removed the prettier config for the angular storybook because most of the code in these stories is not JS or TS, it's actually just a big `template` string.   Whenever I ran prettier on this large string, I would lose a lot of the formatting that I did manually, so I swapped it with an editorconfig file that'll keep our tab indentations consistent at 4 spaces.

Reply:
would this be resolved by splitting the template string into a separate html file?

Reply:
Yes but then the storysource plugin in storybook would be useless because it does not show the most important code. The template is the MVP in our stories shown.

Reply:
If there is a way to preserve the template as a non-string in the same file as what is rendered in the storysource plugin, we can keep prettier.  I don't think it is unreasonable for us to be responsible for manual code indentation since we have the editorconfig to help us.